packages/ERTP/src/types.js

```diff
@@ -312,6 +312,11 @@
  * values labeled with a brand. AmountMath use mathHelpers to do their value arithmetic,
  * and then brand the results, making a new amount.
  *
+ * The MathHelpers are designed to be called only from AmountMath, and so
+ * all methods but coerce can assume their inputs are valid. They only
```

Comment:
Thanks a bunch for documenting this explicitly.

In other parts of code I have had a hard time understanding whether the caller or callee was responsible for runtime validation.